masterlist.yaml

```diff
@@ -382,8 +382,50 @@ plugins:
       - 'ccMTYSSE001-KnightsoftheNine.esl'
   - name: 'Unofficial Skyrim Special Edition Patch.esp'
     global_priority: -120
+    clean:
+      - crc: 0xAE28DBE3
+        util: SSEEdit v3.2.1
     after:
       - 'Lanterns Of Skyrim - All In One - Main.esm'
+  - name: 'BSAssets.esm'
+    clean:
+      - crc: 0x1D0648B6
+        util: SSEEdit v3.2.1
+  - name: 'BSHeartland.esm'
+    dirty:
+    - <<: *dirtyPlugin
```

Comment:
This indentation might work, but it's inconsistent, could you indent by another two spaces?

Reply:
Okay I think its the editor I'm using it's not great for yaml.